desc/magnetic_fields.py

```diff
@@ -1034,6 +1004,238 @@ def compute_magnetic_field(self, coords, params=None, basis="rpz", grid=None):
         return B
 
 
+class DommaschkPotentialField(ScalarPotentialField):
+    """Magnetic field due to a Dommaschk scalar magnetic potential in rpz coordinates.
+
+        From Dommaschk 1986 paper https://doi.org/10.1016/0010-4655(86)90109-8
+
+        this is the field due to the dommaschk potential (eq. 1) for
+        a given set of m,l indices and their corresponding
+        coefficients a_ml, b_ml, c_ml d_ml.
+
+    Parameters
+    ----------
+    ms : 1D array-like of int
+        first indices of V_m_l terms (eq. 12 of reference)
+    ls : 1D array-like of int
+        second indices of V_m_l terms (eq. 12 of reference)
+    a_arr : 1D array-like of float
+        a_m_l coefficients of V_m_l terms, which multiply the cos(m*phi)*D_m_l terms
+    b_arr : 1D array-like of float
+        b_m_l coefficients of V_m_l terms, which multiply the sin(m*phi)*D_m_l terms
+    c_arr : 1D array-like of float
+        c_m_l coefficients of V_m_l terms, which multiply the cos(m*phi)*N_m_l-1 term
+    d_arr : 1D array-like of float
+        d_m_l coefficients of V_m_l terms, which multiply the sin(m*phi)*N_m_l-1 terms
+    B0: float
+        scale strength of the magnetic field's 1/R portion
+
+    """
+
+    def __init__(self, ms, ls, a_arr, b_arr, c_arr, d_arr, B0=1.0):
```

Comment:
Would it be possible to have default values for m,l,a,b,c,d? Like something that would just give a simple 1/R toroidal field?